### DISCUSSION TITLE
tailwind jit

### DISCUSSION THREAD

**Discussion by markfirmware** [alpinejs/alpine#1269] [1. Help]
https://blog.tailwindcss.com/tailwindcss-2-1

Is there an alpinejs example or template that uses the jit feature?

Thanks,
Mark

**Comment by HugoDF** [comment]
We use Tailwind JIT + Alpine in https://github.com/alpine-collective/alpinejs-devtools

**Reply by markfirmware** [reply]
Thanks, I am looking for a minimal example if there is one.

**Reply by HugoDF** [reply]
this is the config: https://github.com/alpine-collective/alpinejs-devtools/blob/master/tailwind.config.js

it should be runnable through the Tailwind CLI tool.

**Reply by markfirmware** [reply]
Thanks - is it possible to have a tw/jit demo in codepen?

**Reply by HugoDF** [reply]
Unlikely, TW JIT needs a Node.js environment to run.

**Reply by markfirmware** [reply]
Thank you. That clarifies things. Tailwind/jit is not jit like alpinejs is jit (alpinejs works at the last possible moment when it is loaded into the browser.)

**Comment by SadWood** [comment]
Hi, i create a demo, hope I can help you
[https://github.com/SadWood/tailwind-jit-alpine.js-vite](url)
